docs/installation/reference/import.rst

```diff
@@ -0,0 +1,193 @@
+Open Zaak exposes several API endpoints (which are not part of the Documenten API standard)
+to help aid the user to import larger amounts of ``EnkelvoudigInformatieObject``'s.
+
+Configuration
+--------------
+
+Environment variables related to the import functionaly are:
+- :ref:`IMPORT_DOCUMENTEN_BASE_DIR <_import_documenten_base_dir>`
+- :ref:`IMPORT_DOCUMENTEN_BATCH_SIZE <_import_documenten_batch_size>`
+
+``IMPORT_DOCUMENTEN_BASE_DIR`` is used to determine the absolute import path for each
+row in the import metadata file. All file paths specified in the import metadata
+file should be relative to the directory specified for this setting.
+By default this is the same directory as the projects directory (``BASE_DIR``) and
+can either be configured through an environment variable or directly through the
+django's settings file being used.
+
+``IMPORT_DOCUMENTEN_BATCH_SIZE`` is the number of rows that will be processed
+at a time.
+
+Process
+-------
+
+The import process consists of several steps with each step having an API endpoint.
+The process consists of the following steps:
+
+1. Creating the ``Import``
+2. Uploading the ``Import`` metadata file
+3. (Optionally) Retrieving the status of the ``Import``
+4. (Optionally) Retrieving the report file of the ``Import``
+
+**Permissions**
+
+Importing ``EnkelvoudigInformatieObjecten`` is not possible for all authenticated users
+for Open Zaak. The reasoning behind this is that not all users, for example have
+the rights to create ``EnkelvoudigInformatieObjecten`` with certain
+``InformatieObjectType``'s. Various permissions exist within Open Zaak and
+therefore to make use of the import functionality, only users with an
+``Applicatie`` with the ``heeft_alle_autorisaties`` set to ``True`` are allowed
+to perform any import operation.
+
+**Creating an Import**
+
+The first step in the import process is that of making an ``Import`` resource.
+This is done through a ``POST`` request. The request body of this request can be empty
+for this step.
+
+Creating an ``Import`` is only possible whenever no other ``Import`` instance exist
+with the statusses ``pending`` or ``active``.
+
+Whenever no other import is pending or active, the endpoint will provide the user
+three URLs: a URL to upload an import metadata file, a URL to retrieve the status
+of an import and another URL for downloading a report of the import. These three
+urls provide the user the ability to progress further in the import process.
+After this request the ``Import`` instance will have it's status changed to
+``pending``.
+
+See the `API documentation for creating an Import`_ for more details.
+
+**Starting an Import**
+
+After creating an `Import` instance, users can upload an import metada file. This
+should be a CSV which consists of rows with data needed to create an
+``EnkelvoudigInformatieObject``. The data this CSV should contain is roughly the
+same as the data needed for creating an ``EnkelvoudigInformatieObject`` through
```

Comment:
```suggestion
should be a CSV which consists of rows with the data needed to create an
``EnkelvoudigInformatieObject``. This data is roughly the
same as the data needed for creating an ``EnkelvoudigInformatieObject`` through
```

I think this sounds more _"proper"_ (not just any data) but not really necessary.

Also an edit to make the next line more concise